apisix/cli/ops.lua

```diff
@@ -153,7 +153,7 @@ local function get_lua_path(conf)
 end
 
 
-local function init(env)
+local function init_sysconf(env)
```

Comment:
can you explain what have you done here?

Reply:
I extracted this method from `local function init(env)` so it can also be used by `local function test_standalone(env)`.

As the information from now in `init_sysconf` is used to [render `apisix.cli.ngx_cli_tpl`](https://github.com/apache/apisix/pull/10630/files#diff-228a51571b53709ad043d1c6c14fc8e508c248a84f8014fe548b5d259cedd5ddR949).